nautobot/core/views/generic.py

```diff
@@ -1041,11 +1024,10 @@ def post(self, request, **kwargs):
 
                             # Update custom fields
                             for field_name in form_custom_fields:
-                                # 2.0 TODO: #824 when we use slug in obj.cf we can just do obj.cf[field_name[3:]]
                                 if field_name in form.nullable_fields and field_name in nullified_fields:
-                                    obj.cf[form_cf_to_key[field_name]] = None
+                                    obj.cf[field_name[3:]] = None
```

Comment:
The previous iteration was a little more readable since it was clear what `form_cf_to_key` was doing and this slicing isn't as obvious. Can you please add a brief comment above this `if/else` stating that `[3:]` is trimming `cf_`?

Reply:
Maybe we can add a classmethod on `CustomField` that's the inverse of the `cf -> "cf_{key}"` function?